src/app/zap-templates/templates/app/CHIPClientCallbacks.zapt

```diff
@@ -3,6 +3,9 @@
 {{#if (chip_has_client_clusters)}}
 #pragma once
 
+#ifndef CHIP_CLIENT_CALLBACKS_H
```

Comment:
headers should use `#pragma once` instead.

especially in generated code, the _H may not be unique

Reply:
I see the pragma once above. Please remove there header guards.

Reply:
I'll fix this next week. Trying to merge the PR before cutoff.